packages/api-aco/src/flp/flp.types.ts

```diff
@@ -5,7 +5,7 @@ import { type AcoContext, type Folder } from "~/types";
 export type FolderAccessLevel = "owner" | "viewer" | "editor" | "public" | "no-access";
 
 export interface FolderPermission {
-    target: string;
+    target: `admin:${string}` | `team:${string}`;
```

Comment:
Can we reuse types from `shared-aco` package here? And I have the same question for `packages/app-aco/src/types.ts`.